keystone/identity/backends/hybrid.py

```diff
@@ -19,7 +19,7 @@
 LDAP_USER_ID_ATTRIBUTE = CONF.ldap.user_id_attribute
 LDAP_USER_TREE_DN = CONF.ldap.user_tree_dn
 LDAP_GENERIC_TREE_DN = CONF.ldap.generic_tree_dn
-LDAP_BUILTIN_USERS = CONF.ldap.builtin_users
+LDAP_BUILTIN_USERS = CONF.ldap.builtin_users.split(',')
```

Comment:
Use ListOpts instead of splitting.

In any case, this has been addressed by PR https://github.com/CiscoSystems/keystone/pull/5 where there is no need for builtin_user custom parameter in keystone config.